site/docs/plugins/keyboard.md

```diff
@@ -260,26 +281,39 @@ await ctx.reply(text, {
 });
 ```
 
-#### Selectively Send Keyboard
+#### Selectively Send Custom Keyboards
 
-You can specify the `selective` option if you want to show the keyboard to only those users that are @-mentioned in the text of the message object, and to the sender of the original message in case your message is a reply.
+You can specify the `selective` option if you want to show the custom keyboard only to those users that are @-mentioned in the text of the message object, and to the sender of the original message in case your message is a [reply](/guide/basics.html#sending-messages-with-reply).
 
 ```ts
 await ctx.reply(text, {
   reply_markup: {
     selective: true,
+    reply_to_message_id: ctx.msg.message_id,
```

Comment:
`reply_to_message_id` is on the main object, not inside `reply_markup`.

Reply:
Good point, thanks.